docs/howtos/envelope-object.md

```diff
@@ -0,0 +1,145 @@
+
+We will now apply the example envelope object above on our hub cluster, the envelope object belongs to a namespace called app hence make sure the namespace app exists on the hub cluster. Then we use a `ClusterResourcePlacement` object to propagate the resource from hub to a member cluster named `kind-cluster-1`
+
+### CRP spec:
+```
+spec:
+  policy:
+    clusterNames:
+    - kind-cluster-1
+    placementType: PickFixed
+  resourceSelectors:
+  - group: ""
+    kind: Namespace
+    name: app
+    version: v1
+  revisionHistoryLimit: 10
+  strategy:
+    type: RollingUpdate
+```
+
+### CRP status:
+
+```
+status:
+  conditions:
+  - lastTransitionTime: "2023-11-30T19:54:13Z"
+    message: found all the clusters needed as specified by the scheduling policy
+    observedGeneration: 2
+    reason: SchedulingPolicyFulfilled
+    status: "True"
+    type: ClusterResourcePlacementScheduled
+  - lastTransitionTime: "2023-11-30T19:54:18Z"
+    message: All 1 cluster(s) are synchronized to the latest resources on the hub
+      cluster
+    observedGeneration: 2
+    reason: SynchronizeSucceeded
+    status: "True"
+    type: ClusterResourcePlacementSynchronized
+  - lastTransitionTime: "2023-11-30T19:54:18Z"
+    message: Successfully applied resources to 1 member clusters
+    observedGeneration: 2
+    reason: ApplySucceeded
+    status: "True"
+    type: ClusterResourcePlacementApplied
+  placementStatuses:
+  - clusterName: kind-cluster-1
+    conditions:
+    - lastTransitionTime: "2023-11-30T19:54:13Z"
+      message: 'Successfully scheduled resources for placement in kind-cluster-1:
+        picked by scheduling policy'
+      observedGeneration: 2
+      reason: ScheduleSucceeded
+      status: "True"
+      type: ResourceScheduled
+    - lastTransitionTime: "2023-11-30T19:54:18Z"
+      message: Successfully Synchronized work(s) for placement
+      observedGeneration: 2
+      reason: WorkSynchronizeSucceeded
+      status: "True"
+      type: WorkSynchronized
+    - lastTransitionTime: "2023-11-30T19:54:18Z"
+      message: Successfully applied resources
+      observedGeneration: 2
+      reason: ApplySucceeded
+      status: "True"
+      type: ResourceApplied
+  selectedResources:
+  - kind: Namespace
+    name: app
+    version: v1
+  - kind: ConfigMap
+    name: envelope-configmap
+    namespace: app
+    version: v1
+```
+
+**Note:**
+In the `selectedResources` section within the `placementStatus` for `kind-cluster-1` we only show that the envelope object got propagated, we don't include all the resources within the envelope object in the status.
+
+From the `selectedResources` section within the `placementStatus` for `kind-cluster-1` we see that the namespace `app` along with the configmap `envelope-configmap` got propagated. And the user can also verify with resources mentioned within in the `envelope-configmap` object are also propagated in this case a `ResourceQuota` object and `MutatingWebhookConfigurations` object.
```

Comment:
Please give an example when envelop objects failed to apply. The status is different from others.